triggering.go

```diff
@@ -188,49 +206,41 @@ func (dsp *DataStreamProcessor) autoTriggerComputeAppend(records []*DataRecord)
 }
 
 // TriggerData analyzes a DataSegment to find and generate triggered records.
-// All edge triggers are found, then level triggers, then auto and noise triggers.
+// All edge-multitriggers are found, OR [all edge triggers are found, then level
+// triggers, then auto, and noise triggers].
 // Returns slice of complete DataRecord objects, while dsp.lastTrigList stores
 // a triggerList object just when the triggers happened.
 func (dsp *DataStreamProcessor) TriggerData() (records []*DataRecord) {
-	if dsp.EdgeMulti {
-		records = dsp.edgeMultiTriggerComputeAppend(records)
-		trigList := triggerList{channelIndex: dsp.channelIndex}
-		trigList.frames = make([]FrameIndex, len(records))
-		for i, r := range records {
-			trigList.frames[i] = r.trigFrame
-		}
-		trigList.keyFrame = dsp.stream.DataSegment.firstFrameIndex
-		trigList.keyTime = dsp.stream.DataSegment.firstTime
-		trigList.sampleRate = dsp.SampleRate
-		trigList.firstFrameThatCannotTrigger = dsp.stream.DataSegment.firstFrameIndex +
-			FrameIndex(len(dsp.stream.rawData)) - FrameIndex(dsp.NSamples-dsp.NPresamples)
-
-		dsp.lastTrigList = trigList
-		// EdgeMulti does not play nice with other triggers, so return now!!
-		return records
-	}
```

Comment:
To be clear, the block above is completely redundant from lines 196-209. That was done to make it safe to return, but I didn't like the duplication. I simplified the logic, in my opinion.